src/bare_americana.js

```diff
@@ -2,16 +2,20 @@
 
 import "maplibre-gl/dist/maplibre-gl.css";
 
-import { createMap, loadRTLPlugin } from "./js/map_builder.js";
+import { createMap, loadRTLPlugin, buildStyle } from "./js/map_builder.js";
 
 loadRTLPlugin();
 
-export const map = createMap(
-  window,
-  (shields) => shieldDefLoad(),
-  [-94, 40.5],
-  4
-);
+export const map = createMap(window, (shields) => shieldDefLoad(), {
+  container: "map", // container id
+  hash: "map",
+  antialias: true,
+  style: buildStyle(),
+  center: [-94, 40.5],
+  zoom: 4,
+  fadeDuration: 0,
```

Comment:
The default crossfade is nice when switching languages. Label placement sometimes differs dramatically from one zoom level to another; the fade duration mitigates that effect. We should get to the bottom of why you were seeing the hung crossfade so we can reenable that functionality.

Reply:
The cross fade isn't hung, it just takes up unnecessary time in a headless utility use case. The fade is only disabled for the purpose of generating render samples.

Reply:
The docs say

> fade-in/fade-out animation for label collisions **after** initial map load

I imagine it's load, 300ms, idle.

If the "bare" map page is useful for something other than driving the screenshot script then a more flexible setup would be needed.

Reply:
Ah, I misunderstood. This is perfectly reasonable for a headless setup. That said, we probably should still add the attribution no matter what. It can be smaller if necessary.